esmvalcore/_config/extra_facets/ipslcm-mappings.yml

```diff
@@ -0,0 +1,239 @@
+    uas: {ipsl_varname: u10m, <<: *atmvars}
+    "vas": {ipsl_varname: v10m, <<: *atmvars}
+    sfcWind: {ipsl_varname: wind10m, <<: *atmvars}
+    hurs: {ipsl_varname: rh2m, <<: *atmvars}
+
+    # ATM general variables
+    precip: {ipsl_varname: precip, <<: *atmvars}
+    prw: {ipsl_varname: prw, <<: *atmvars}
+    slp: {ipsl_varname: slp, <<: *atmvars}
+    t2m: {ipsl_varname: t2m, <<: *atmvars}
+    q2m: {ipsl_varname: q2m, <<: *atmvars}
+    u10m: {ipsl_varname: u10m, <<: *atmvars}
+    v10m: {ipsl_varname: v10m, <<: *atmvars}
+    wind10m: {ipsl_varname: wind10m, <<: *atmvars}
+
+    #  -> Turbulent fluxes
+    taux: {ipsl_varname: taux, <<: *atmvars}
+    tauy: {ipsl_varname: tauy, <<: *atmvars}
+
+    #  -> Radiative down at TOA
+    SWdnTOA: {ipsl_varname: SWdnTOA, <<: *atmvars}
+
+    #  -> Radiative down at TOA
+    topl: {ipsl_varname: topl, <<: *atmvars}
+    SWupTOA: {ipsl_varname: SWupTOA, <<: *atmvars}
+    topl0: {ipsl_varname: topl0, <<: *atmvars}
+    SWupTOAclr: {ipsl_varname: SWupTOAclr, <<: *atmvars}
+
+    #  -> Radiative up at Surface
+    LWupSFC: {ipsl_varname: LWupSFC, <<: *atmvars}
+    SWupSFC: {ipsl_varname: SWupSFC, <<: *atmvars}
+    SWupSFcclr: {ipsl_varname: SWupSFcclr, <<: *atmvars}
+    LWupSFcclr: {ipsl_varname: LWupSFcclr, <<: *atmvars}
+
+    #  -> Radiative down at Surface
+    LWdnSFC: {ipsl_varname: LWdnSFC, <<: *atmvars}
+    SWdnSFC: {ipsl_varname: SWdnSFC, <<: *atmvars}
+    LWdnSFcclr: {ipsl_varname: LWdnSFcclr, <<: *atmvars}
+    SWdnSFcclr: {ipsl_varname: SWdnSFcclr, <<: *atmvars}
+
+    #  -> Turbulent fluxes
+    hfls: {ipsl_varname: flat, <<: *atmvars, positive: down}
+    hfss: {ipsl_varname: sens, <<: *atmvars, positive: down}
+    tauu: {ipsl_varname: taux, <<: *atmvars}
+    tauv: {ipsl_varname: tauy, <<: *atmvars}
+
+    #  -> Clouds
+    clt: {ipsl_varname: cldt, <<: *atmvars}
+
+    #  -> Radiative up at TOA
+    rlut: {ipsl_varname: topl, <<: *atmvars}
+    rsut: {ipsl_varname: SWupTOA, <<: *atmvars}
+    rlutcs: {ipsl_varname: topl0, <<: *atmvars}
+    rsutcs: {ipsl_varname: SWupTOAclr, <<: *atmvars}
+
+    #  -> Radiative down at TOA
+    rsdt: {ipsl_varname: SWdnTOA, <<: *atmvars}
+
+    #  -> Radiative up at Surface
+    rlus: {ipsl_varname: LWupSFC, <<: *atmvars}
+    rsus: {ipsl_varname: SWupSFC, <<: *atmvars}
+    rsuscs: {ipsl_varname: SWupSFcclr, <<: *atmvars}
+    rluscs: {ipsl_varname: LWupSFcclr, <<: *atmvars}
+
+    #  -> Radiative down at Surface
+    rlds: {ipsl_varname: LWdnSFC, <<: *atmvars}
+    rsds: {ipsl_varname: SWdnSFC, <<: *atmvars}
+    rldscs: {ipsl_varname: LWdnSFcclr, <<: *atmvars}
+    rsdscs: {ipsl_varname: SWdnSFcclr, <<: *atmvars}
+
+  # ========================================================================================
+  Lmon:
+    # ========================================================================================
+    # SRF -- Land surface - ORCHIDEE
+
+    #  --> !!! This will not stay in the param file !!! --------------------
```

Comment:
What is meant here by param file? This file? Or a particular IPSL file? When will this be removed?

Reply:
Discarded. It arrived there by an automated process and I didn't re-read as carefully as you did